### PR TITLE
Added all Contao repos

### PR DESCRIPTION
**Pull request by aschempp**

### CODE REVIEW

File: meta/contao/mobile-page-layout-bundle/de.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+de:
+    title: Mobiles Seitenlayout für Contao 4.8+
+    description: >
+        In Contao 4.8 wurde die Unterstützung für mobile Seitenlayouts entfernt.
+        Wenn ihre Webseite diese Funktion weiterhin benötigt, installieren Sie diese Erweiterung. Beachten Sie
```
Comment (xchs) [inline meta/contao/mobile-page-layout-bundle/de.yml]: Typo: **i**hre -> **I**hre 
(Sollten wir hier vielleicht ebenfalls bei der Du-Form bleiben, wie in der [README](https://github.com/contao/package-metadata/blob/master/docs/de/README.md) empfohlen?)

Typo: Nach "Beachten Sie" kommt ein Komma.

File: meta/contao/official-demo/en.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+en:
+    title: Official Demo Theme for Contao
+    description: This is the official demo theme for Contao.
```
Comment (xchs) [inline meta/contao/official-demo/en.yml]: Im deutschen Beschreibungstext steht "Contao 4", hier "Contao".

File: meta/contao/manager-bundle/en.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+en:
+    title: Contao 4 Managed Edition
+    description: This bundle contains code necessary to run the Contao 4 Managed Edition.
```
Comment (leofeyer) [inline meta/contao/manager-bundle/en.yml]: Should be synchronized with https://github.com/contao/contao/pull/808

File: meta/contao/manager-bundle/de.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+de:
+    title: Contao 4 Managed Edition
+    description: Dieses Paket enthält alle nötigen Bibliotheken für die Contao 4 Managed Edition.
```
Comment (leofeyer) [inline meta/contao/manager-bundle/de.yml]: Should be synchronized with https://github.com/contao/contao/pull/808

File: meta/contao/mobile-page-layout-bundle/de.yml (hunk 1 above)
Comment (richardhj) [reply meta/contao/mobile-page-layout-bundle/de.yml]: Sollte "Du" sein. Siehe 36cea8b1843fc2acfdfd6a5b66f6ca408bd78883.

**Comment by aschempp** [comment]
> Why do you add translations for non-discoverable packages?

Contao Manager 1.2 shows information for all packages. Discoverable means they can be searched for, but suggestions etc. are not found in the search but can have a package description.

File: meta/contao/mobile-page-layout-bundle/de.yml [hunk 5]
```diff
@@ -2,5 +2,5 @@ de:
     title: Mobiles Seitenlayout für Contao 4.8+
     description: >
         In Contao 4.8 wurde die Unterstützung für mobile Seitenlayouts entfernt.
-        Wenn ihre Webseite diese Funktion weiterhin benötigt, installieren Sie diese Erweiterung. Beachten Sie
+        Wenn deine Webseite diese Funktion weiterhin benötigt, installiere diese Erweiterung. Beachte
```
Comment (xchs) [inline meta/contao/mobile-page-layout-bundle/de.yml]: Nach "_Beachte_" käme ein Komma.

File: meta/contao/official-demo/en.yml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+en:
+    title: Official Demo Theme for Contao
```
Comment (leofeyer) [inline meta/contao/official-demo/en.yml]:
```suggestion
    title: Official demo theme for Contao
```

File: meta/contao/skeleton-bundle/en.yml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+en:
+    title: Skeleton Bundle for Contao
```
Comment (leofeyer) [inline meta/contao/skeleton-bundle/en.yml]:
```suggestion
    title: Skeleton bundle for Contao
```

File: meta/contao/skeleton-bundle/en.yml [hunk 8]
```diff
@@ -0,0 +1,5 @@
...
+    description: >
+        This bundle adds rapid development tools and code generators to Contao 4.
```
Comment (leofeyer) [inline meta/contao/skeleton-bundle/en.yml]: It actually does neither of that. 🤷‍♂

File: meta/contao/skeleton-bundle/en.yml [hunk 9]
```diff
@@ -0,0 +1,5 @@
...
+        These features are only useful to developers, so don't use it in production apps!
```
Comment (leofeyer) [inline meta/contao/skeleton-bundle/en.yml]:
```suggestion
        These features are only useful for developers, so don't use it in production apps!
```

File: meta/contao/tcpdf-bundle/en.yml [hunk 10]
```diff
@@ -0,0 +1,3 @@
+en:
+    title: PDF Articles for Contao
```
Comment (leofeyer) [inline meta/contao/tcpdf-bundle/en.yml]:
```suggestion
    title: PDF articles for Contao
```

File: meta/contao/skeleton-bundle/en.yml (hunk 8 above)
Comment (aschempp) [reply meta/contao/skeleton-bundle/en.yml]: yeah, I thought this is the maker bundle 😂

**Comment by leofeyer** [comment]
As discussed with @aschempp in Slack, "Contao Managed Edition" is eventually a product name and thus needs to be written as "Contao Managed Edition" in all languages.

File: meta/contao/official-demo/de.yml [hunk 11]
```diff
@@ -0,0 +1,3 @@
+de:
+    title: Offizielles Demo-Theme für Contao
```
Comment (leofeyer) [inline meta/contao/official-demo/de.yml]:
```suggestion
    title: Contao Official Demo
```

**Pull request merged**